### DISCUSSION TITLE
Should I use the `httprpc` or `rpc` plugin?

### DISCUSSION THREAD

**Discussion by blueberryapple** [Novik/ruTorrent#2439] [Q&A] [answered]
I'm confused on which is more resource heavy?

**Comment by stickz** [comment, marked as answer]
`httprpc` is always better, unless you're hosting a raspberry pi or something.

**Reply by blueberryapple** [reply]
Thanks, I actually am hosting on a raspberry pi

**Reply by MarkusLange** [reply]
Btw. works flawless with httprpc inside an ESXI server on an raspberry pi4 (8GB) (round about 300 torrents)

**Comment by MarkusLange** [comment]
I have an additional question on my setup both are loaded and active so both are not interfering each other?
